corehq/apps/accounting/invoicing.py

```diff
@@ -566,8 +594,24 @@ def unit_cost(self):
     def quantity(self):
         if self.is_prorated:
             return self.num_prorated_days
+        if self.invoice.is_customer_invoice:
+            if self.invoice.account.invoicing_plan == InvoicingPlan.QUARTERLY:
+                return self.quantity_over_period(3)
+            elif self.invoice.account.invoicing_plan == InvoicingPlan.YEARLY:
+                return self.quantity_over_period(12)
         return 1
 
+    def quantity_over_period(self, num_months):
```

Comment:
I'm having trouble following what this function does.  I'd recommend adding a comment and/or making the code more self-documenting.